curriculum/challenges/english/09-information-security/information-security-with-helmetjs/ask-browsers-to-access-your-site-via-https-only-with-helmet.hsts.md

```diff
@@ -16,7 +16,7 @@ HTTP Strict Transport Security (HSTS) is a web security policy which helps to pr
 
 Configure `helmet.hsts()` to use HTTPS for the next 90 days. Pass the config object `{maxAge: timeInSeconds, force: true}`. You can create a variable `ninetyDaysInSeconds = 90*24*60*60;` to use for the `timeInSeconds`. Replit already has hsts enabled. To override its settings you need to set the field "force" to true in the config object. We will intercept and restore the Replit header, after inspecting it for testing.
 
-Note: Configuring HTTPS on a custom website requires the acquisition of a domain, and a SSL/TLS Certificate.
```

Comment:
Here I guess the article depends on if you read it extended or not, if you read "Secure" or "S". What's most common I wonder

Reply:
I'm a native English speaker and I can't think of a context where "a SSL certificate" doesn't sound weird

Reply:
"a S(ecure)S(ocket)L(ayer)"

Reply:
Oh I see what you mean, I was only thinking of it as the abbreviation.